### DISCUSSION TITLE
Google button data-type

### DISCUSSION THREAD

**Discussion by coyzlimit** [abacritt/angularx-social-login#552] [Q&A] [answered]
How to set the Google button data-type to standard?

**Comment by josephfo** [comment, marked as answer]
From the docs...
![image](https://user-images.githubusercontent.com/8108665/181839901-761425ac-fe1c-46e8-b439-c0a5728ed672.png)

**Reply by cristi200915** [reply]
![image](https://user-images.githubusercontent.com/5085449/183001658-ff378541-72ad-47f2-bcd2-31b79a0aa416.png)
I set type to standard and still showing icon type

**Reply by mfrizqi** [reply]
Me too, where did i miss?
